clash-cores/src/Clash/Cores/Floating/Xilinx/BlackBoxes.hs

```diff
@@ -0,0 +1,120 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE RecordWildCards #-}
+module Clash.Cores.Floating.Xilinx.BlackBoxes
+  ( addFloatTclTF
+  ) where
+
+import Prelude
+
+import Control.Monad.State (State())
```

Comment:
You don't need the extra parens after `State`, right?

Reply:
Right